elasticdl_client/common/k8s_resource.py

```diff
@@ -84,6 +84,9 @@ def parse(resource_str):
                     % k
                 )
             _valid_gpu_spec(v)
+        elif "nvidia.com" in k:
```

Comment:
k.startswith("nvidia.com") will be more precise

Reply:
Yeah！